src/components/organisms/TimingEventModal/TimingEventModal.tsx

```diff
@@ -246,26 +243,25 @@ export const TimingEventModal: React.FC<TimingEventModalProps> = ({
                 </div>
               </div>
 
-              <AdminSection title="Duration">
-                <div className="TimingEventModal__container">
-                  <label>
-                    <input
-                      name="duration_hours"
-                      className="input-group__modal-input input-group__modal-input--indent"
-                      placeholder="hours"
-                      ref={register}
-                    />
-                    hour(s)
-                  </label>
-                  <label>
-                    <input
-                      name="duration_minutes"
-                      className="input-group__modal-input input-group__modal-input--indent"
-                      placeholder="minutes"
-                      ref={register}
-                    />
-                    minutes
-                  </label>
+              <div className="input-group">
+                <label>Duration</label>
+                <div className="TimingEventModal__duration_container">
+                  <input
```

Comment:
We have `AdminInput` component which should fit fine and it's a general use component that we're reusing in admin v3

Reply:
Had this discussion with Goran. I would *love* to use it. Doing so requires that I update a load of AV1 code due to field names not being what `AdminInput` expects. For now, we agreed to keep it as is.

Reply:
`<div className="input-group">` is still an issue for me. I'd simply leave the old section in place.